src/components/haptic-feedback.js

```diff
@@ -0,0 +1,57 @@
+const strengthForIntensity = {
+  low: 0.07,
+  medium: 0.2,
+  high: 1
+};
+
+AFRAME.registerComponent("haptic-feedback", {
+  schema: {
+    hapticEventName: { default: "haptic_pulse" }
+  },
+
+  init: function() {
+    this.pulse = this.pulse.bind(this);
+    this.getActuator = this.getActuator.bind(this);
+    this.getActuator().then(actuator => {
+      this.actuator = actuator;
+    });
+  },
+
+  getActuator() {
+    return new Promise((resolve, reject) => {
```

Comment:
Still really not a fan of this but if we cant just listen to `controllerconnected` for some reason I guess this will have to do...